test/integration/integration_test_fixtures.py

```diff
@@ -297,6 +301,25 @@ def getTestServerRootUri(self):
     return super(HttpIntegrationTestBase, self).getTestServerRootUri(False)
 
 
+class HttpIntegrationTestBaseWithV2Bootstrap(IntegrationTestBase):
+  """Base for running plain http tests against the Nighthawk test server.
+
+  NOTE: any script that consumes derivations of this, needs to also explicitly
+  import server_config, to avoid errors caused by the server_config not being found
+  by pytest.
+  """
+
+  def __init__(self, ip_version, server_config):
+    """See base class."""
+    super(HttpIntegrationTestBaseWithV2Bootstrap, self).__init__(ip_version,
+                                                                 server_config,
+                                                                 bootstrap_version_arg="2")
```

Comment:
This actually looks like we are making it a string containing a number, can we use an int if we carry a numerical value?